### PR TITLE
added tourists part

### PR DESCRIPTION
**Pull request by aratitanyan**

### CODE REVIEW

File: src/components/user/tour/tour.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import Tourists from "./tourists/Tourists";
+
 
 export default function Tour() {
-  return <Typography paragraph>The 'Tour' Page comming soon...</Typography>;
+
+    return (
+        <Tourists></Tourists>
```
Comment (sasuntitan) [inline src/components/user/tour/tour.js]: please make component self closing

File: package.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "dependencies": {
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
+    "@date-io/date-fns": "v1",
```
Comment (sasuntitan) [inline package.json]: reinstall with 1st version

File: src/components/user/tour/tourists/TouristForm.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        "& .MuiInputBase-root": {
+          width: "40",
+          margin: theme.spacing(1)
+        }
+      }
+    }
+  )
+);
+
+const initialFormValues = {
```
Comment (sasuntitan) [inline src/components/user/tour/tourists/TouristForm.js]: integrate formik with yup validator

File: src/components/user/tour/tourists/Tourists.js [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+
+  const [records, setRecords] = useState(touristService.getAllTourists());
+  const [recordForEdit, setRecordForEdit] = useState(null);
+  const [filterFn, setFilterFn] = useState({
+    fn: items => {
+      return items;
+    }
+  });
+  const pages = [5, 10, 15];
```
Comment (sasuntitan) [inline src/components/user/tour/tourists/Tourists.js]: use useRef

File: src/components/user/tour/tourists/TouristForm.js [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+                            <div>
+                                <Button
+                                    type="submit"
+                                    color="primary"
+                                    variant="contained"
+                                    onClick={handleSubmit}
+                                >Submit</Button>
+                                <Button
+                                    color="default"
+                                    onClick={() => resetForm(
```
Comment (sasuntitan) [inline src/components/user/tour/tourists/TouristForm.js]: don't use inline functions in templates

**Review by sasuntitan — APPROVED** [review]

**Review by haykghazaryan-st — APPROVED** [review]

**Pull request merged**